deploy/cephcsi/image/Dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM ceph/ceph:v14.2
+FROM ceph/ceph:v14.2.2-20190826
```

Comment:
`ceph/ceph:v14.2` is being updated as latest 14.2.x version, but might also see new builds for OS package updates. It is important to have the latest (security/bugfix) updates included in new builds. With that, I do not approve of requiring a fixed (outdated) base image.

Reply:
(clarification required) So when we build the CSI image, stating the FROM tag, does a download of the CSI image to a running cluster, pick the latest base image (IOW the FROM tag), or is the base FROM image baked into the CSI image and needs a rebuild of CSI image to catch up to the latest?

If the former (auto updated), I can see how latest 14.2 can always be picked up by installations running the CSI image, assuming updated to base image is somehow tracked and understood by the system.

If the latter (CSI image needs to be rebuilt), then we need to rebuild CSI image each time the base image is updated for (almost) any reason.

So, depending on 14.2 image version is fine, but I suspect we need to rebuild CSI images whenever 14.2 changes to keep our image contents updated.

This is a request for clarification as I am not well versed in how this is processed when we build our images. Based on the answer, we either need a rebuild "policy" or depend on a specific version and move it higher when we rebuild.